npe2/_inspection/_fetch.py

```diff
@@ -0,0 +1,344 @@
+from __future__ import annotations
```

Comment:
i don't know why git didn't pick it up, but most of this module is just moved from npe2/_fetch